main.ts

```diff
@@ -589,15 +607,24 @@ function renderFinances(options) {
 }
 
 function renderHealth(options) {
-  renderHorizontalBarChat(d3.select('#sickness'), SICK, options.width, 250, false);
-  renderPieChart(d3.select('#health-insurance'), OHIP, options.width * 0.75, options.width * 0.75);
+  renderHorizontalBarChat(d3.select('#hospital'), HOSPITAL, options.width, 250, false);
+  renderHorizontalBarChat(d3.select('#health-extension'), EXTENSION_DUE_ILLNESS, options.width, 250, false);
+  renderHorizontalBarChat(d3.select('#sick-in-class'), SICK_IN_CLASS, options.width, 250, false);
+  // renderPieChart(d3.select('#health-extension'), EXTENSION_DUE_ILLNESS, options.width * 0.75, options.width * 0.75);
   renderPieChart(d3.select('#mental-health-count'), MENTAL_HEALTH, options.width * 0.75, options.width * 0.75);
   renderHorizontalBarChat(d3.select('#mental-health-issues'), MENTAL_HEALTH_ISSUES, options.width, 250, false);
+  renderHorizontalBarChat(d3.select('#mental-health-terms'), MH_AFFECTED_TERMS, options.width, 250, false);
+  renderHorizontalBarChat(d3.select('#councelling'), COUNSELLING_SERVICES, options.width, 250, false);
```

Comment:
Ohhhhhhh I thought the incorrect spelling would cause problems. You can ignore my suggestion

Reply:
LOL no I just cannot spell to save my life